mobile/ios/Interface.h

```diff
@@ -28,7 +28,8 @@
                  essenceLength:(NSNumber*)essenceLength
                          count:(NSNumber*)count
                         nprocs:(NSNumber*)nprocs
-               miningThreshold:(NSNumber*)miningThreshold;
+               miningThreshold:(NSNumber*)miningThreshold
+                   fullySecure:(BOOL*)fullySecure;
```

Comment:
```suggestion
                   fullySecure:(BOOL)fullySecure;
```
I don't think we need to use pointers here since [`BOOL`](https://developer.apple.com/documentation/objectivec/bool) is just a typedef for `bool` (64-bit iOS) or `signed char` (macOS and 32-bit iOS)